buildSrc/subprojects/profiling/src/main/kotlin/org/gradle/gradlebuild/profiling/buildscan/BuildScanPlugin.kt

```diff
@@ -65,11 +64,9 @@ open class BuildScanPlugin : Plugin<Project> {
     override fun apply(project: Project): Unit = project.run {
         buildScan = the()
 
-        extractCiOrLocalData()
-        extractVcsData()
+        extractCIData()
 
         if (isCiServer) {
-            doNotUploadInBackground()
```

Comment:
Why was this removed?

Reply:
The logic is moved to plugin code: https://github.com/gradle/gradle-enterprise-conventions-plugin/blob/9e0fc6545148014273ef2fd3e1c94beacfb7fb54/src/main/java/com/gradle/enterprise/conventions/GradleEnterpriseConventionsPlugin.java#L67